statchatexpress/models/user.js

```diff
@@ -0,0 +1,18 @@
+import { DataTypes } from 'sequelize';
+import { sequelize } from '../database.js';
+
+export const User = sequelize.define('User', {
+  Username: {
+    type: DataTypes.STRING,
+    allowNull: false
```

Comment:
Do we want to allow multiple people with the same username?

Reply:
nope, i will change this.